credentialdigger/client.py

```diff
@@ -8,6 +8,7 @@
 
 import yaml
 from github import Github
+from git import GitCommandError
```

Comment:
Please follow alphabetical order of imports (you can use [`isort`](https://pycqa.github.io/isort/) if you don't want to do it manually)